services/requester/requester.go

```diff
@@ -125,13 +132,38 @@ func NewEVM(
 		)
 	}
 
+	head := &types.Header{
+		Number:   big.NewInt(20_182_324),
+		Time:     uint64(time.Now().Unix()),
+		GasLimit: 30_000_000,
+	}
+	emulatorConfig := emulator.NewConfig(
+		emulator.WithChainID(config.EVMNetworkID),
+		emulator.WithBlockNumber(head.Number),
+		emulator.WithBlockTime(head.Time),
+	)
+	evmSigner := emulator.GetSigner(emulatorConfig)
+	validationOptions := &txpool.ValidationOptions{
+		Config: emulatorConfig.ChainConfig,
```

Comment:
Why not use https://github.com/onflow/flow-go/blob/fc7c03c2c524fa8881bc9e3b4cf53b6b4f471dc3/fvm/evm/emulator/config.go#L51

Reply:
It was suggested by Ramtin (https://github.com/onflow/flow-evm-gateway/pull/323#discussion_r1661452896), to construct a new config, with the desired options applied. This also facilitates fetching the `signer`, based on the created config. Note that we do not need the default config for all networks, as the default config is hardcoded to PreviewNet.